### ISSUE TITLE
Netflix WidevineCdm

### ISSUE THREAD

**Issue by LutzPS** [jiahaog/nativefier#225]
I can make a package with WidevineCdm?

**Comment by Silent-Hunter** [comment]
I too would like to know if Widevine is supported.

**Comment by jiahaog** [comment]
This is a duplicate of #113! It's not supported now, but feel free to submit a pull request!

**Comment by quanglam2807** [comment]
@LutzPS  @Silent-Hunter  It has been fixed in WebCatalog webcatalog/desktop#4

**Comment by mikhoul** [comment]
I'm new with nativefier and I had the same problem with  Spotify, browsing the issue **I think it was fixed** but is have to be approved before going in the main branch: https://github.com/jiahaog/nativefier/pull/440 

Sorry I'm a newbie with github technical terms,

Regards :octocat:
